landoapi/api/transplants.py

```diff
@@ -180,8 +182,10 @@ def _assess_transplant_request(phab, landing_path):
         reviewers,
         users,
         projects,
-        get_secure_project_phid(phab),
+        is_secure,
```

Comment:
You've changed the semantics here. Previously we were checking the security of each revision, but now this would check only a single revision and apply it's security status to all the revisions.

Please revert things to just pass the `phid` and let the warning system check each revision. Having the new `revision_is_secure()` function will still simplify the warning body a lot.

Reply:
I added an integration test to catch the bug and reverted the call chain to the old behaviour.